### PR TITLE
Fill caseposneg test case field

### PR DESCRIPTION
**Pull request by elyezer**
Update the value of caseposneg according to the presence of negative in
the test name. If negative is present on test name then the caseposneg
is going to be negative otherwise it is going to be positive.

### CODE REVIEW

File: betelgeuse.py [hunk 1]
```diff
@@ -209,12 +209,18 @@ def add_test_case(args):
         # Is the test automated? Acceptable values are:
         # automated, manualonly, and notautomated
         auto_status = 'automated' if test.automated else 'notautomated'
+        caseposneg = 'negative' if 'negative' in test.name else 'positive'
```
Comment (sthirugn) [inline betelgeuse.py:212]: can you use a more meaningful name instead of `caseposneg` please?

Comment (sthirugn) [inline betelgeuse.py:212]: perhaps something as easy as case_pos_neg? :)

**Comment by omaciel** [comment]
ACK pending resolution for @sthirugn comment.

**Comment by elyezer** [comment]
@omaciel @sthirugn this is the name of the custom field on polarion. That said instead of having one name for the variable and the other for the custom field I prefer to have same for both. Since I can't change the custom field name I need to follow it.

Edit: I'm also looking for changing the auto_status to the same as expected on the custom field, for the same reason.

**Comment by sthirugn** [comment]
Its okay if our variable name does not match the api field in my opinion.  As we all know, since python does not have data types defined in the code, the beauty of the python code relies heavily on the meaningful variable names and function names only.
But that said, @elyezer this is not a blocker for this PR so I am okay with merging this as is.

**Pull request merged**